pubspec.yaml

```diff
@@ -1,7 +1,8 @@
 name: markdown
 version: 2.1.8
 
-description: A library for converting markdown to HTML.
+description: A portable Markdown library written in Dart that can parse
+ Markdown into HTML on both the client and server.
```

Comment:
I'm not crazy about "on both the client and server," as I don't think those are defined well... are these words required to get the letter count up?

Reply:
@srawlins No, they aren't needed. To pass the package description length requirement the description needs to be between 60 and 180 characters. If the text "on both the client and server" is removed the description will be 79 characters in length which will meet the requirement. I'll modify the description text and submit the change.

BTW, the expanded description I submitted was taken from the first line of the README.md file:
> A portable Markdown library written in Dart. It can parse Markdown into HTML on both the client
> and server.

If this statement is incorrect or misleading the README.md should probably be updated at some point.

Reply:
Haha, that's interesting. I suspect not written by me. In any case, I love the change.